src/containerapp/azext_containerapp/_params.py

```diff
@@ -389,3 +391,7 @@ def load_arguments(self, _):
         c.argument('registry_pass', validator=validate_registry_pass, options_list=['--registry-password'], help="The password to log in to container registry. If stored as a secret, value must start with \'secretref:\' followed by the secret name.")
         c.argument('registry_user', validator=validate_registry_user, options_list=['--registry-username'], help="The username to log in to container registry.")
 
+    with self.argument_context('containerapp java logger') as c:
+        c.argument('logger_name', help="The logger name.")
+        c.argument('logger_level', arg_type=get_enum_type(["off", "error", "info", "debug", "trace", "warn"]), help="Set the log level for the specific logger name. Allowed values: off, error, info, debug, trace, warn.")
```

Comment:
You already has get_enum_type, that is good.
You don't need `Allowed values: off, error, info, debug, trace, warn` in the message because they are redundent, you can test it with `--help`